test/helpers/mentor_dashboard_helper_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class MentorDashboardHelperTest < ActionView::TestCase
+  test 'time_ago_in_days calculates time in days' do
+    two_days_ago = Date.today - 38.days
+    assert_equal '38 days', time_ago_in_days(two_days_ago)
```

Comment:
This is very confusing. The local variable is named `two_days_ago`, and there's a magic number `38`. Why 38? How is two days ago `38 days`?

As a rule of thumb, I use the simplest, stupidest values possible that will demonstrate that the behavior I want is correct. I also want to check boundary values.

So for this behavior I would want to check at least 1 and 2 (because of pluralization), and possibly 0 if that would say `0 days` since if we happened to not use the default Rails behavior here, we might end up with `0 day`.